tutorials/data-science-extraction/index.md

```diff
@@ -0,0 +1,390 @@
+---
+title: Data extraction from audio and text
+description: Extract structured data from audio and text.
+author: jerjou
+tags: Data Science
+date_published: 2017-05-10
+---
+Oftentimes, the raw data you've gathered is not in a form that is directly
+explorable using the data exploration tools at your disposal. Making it usable
+may require converting the format, extracting the information type you're
+seeking, or adding metadata to further structure the data.
+
+Google Cloud Platform makes it easy to write specialized functions to transform
+your data and chain them into a [pipeline](preprocessing.md), and provides a
+number of Machine Learning APIs that enable you to [transcribe audio][speech];
+[identify faces, landmarks, and text][vision] in images; [translate][translate]
+between languages; and [provide structure to prose][natural-language].
+
+In this tutorial, you'll write several functions to perform various
+transformations and extraction to turn raw audio files into structured,
+queryable data. These functions can then be easily combined together into a
+reusable data ingestion pipeline, as described in the
+[preprocessing](preprocessing.md) tutorial.
+
+[speech]: /speech
+[vision]: /vision
+[translate]: /translate
+[natural-language]: /natural-language
+
+### Objectives
+
+This data extraction pipeline example can be described as a series of discrete
+steps:
+
+* Unzipping a collection of `mp3` audio files
+* Converting the `mp3`s to raw `LINEAR16` audio
+* Uploading the raw audio files to Google Cloud Storage
+* Extract transcription of speech from the audio files
+* Process the transcription to identify the entities.
+* Save the results
+
+### Prerequisites
+
+* You've enabled the [Cloud Speech API][speech-api] and the [Cloud Natural
+  Language API][nl-api] in your Cloud Console project.
+* You've downloaded [service account credentials][service-account] and set the
+  `GOOGLE_APPLICATION_DEFAULT` environment variable to point to them. ([More
+  details][auth])
+* You have basic familiarity with [Python][python] programming.
+* You've installed [pip][pip].
+* You've installed [virtualenv][virtualenv]
+
+[speech-api]: https://console.cloud.google.com/apis/api/speech.googleapis.com/overview?project=_
+[nl-api]: https://console.cloud.google.com/apis/api/language.googleapis.com/overview?project=_
+[service-account]: https://console.cloud.google.com/apis/credentials?project=_
+[python]: https://www.python.org/
+[pip]: https://pip.pypa.io/en/latest/installing/
+[virtualenv]: http://virtualenv.pypa.io
+[auth]: /docs/authentication#developer_workflow
+
+## Download the tutorial data
+
+For this tutorial, we'll extract data from readings of Aesop's Fables from
```

Comment:
Ideally, this whole document would be switched out of first person plural ("Now we'll do the thing") to the second person singular ("Now you'll do the thing"). It would be even better to avoid the future tense, especially in instructions. So "Next, you'll do the thing" would become "Next, do the thing" (this makes instructions imperative, which reads more like a call to action). Neither item is a show stopper, though.